pubspec.yaml

```diff
@@ -5,7 +5,7 @@ publish_to: none
 
 environment:
   sdk: '>=2.18.5 <3.0.0'
-  flutter: 3.3.9
+  flutter: 3.7.0
```

Comment:
we want to keep 3.3.9 for now

Reply:
This is a requirement for `fetch_client`: https://github.com/dart-lang/http/issues/595#issuecomment-1399608094

(flutter version ... dart version)
<img width="541" alt="Screenshot 2023-03-22 at 8 11 22 AM" src="https://user-images.githubusercontent.com/7699058/226769532-665a9dcc-ee51-42be-93c4-3cc0354f83ae.png">